### PR TITLE
Add Katacoda Embedded Live Environment

### PR DESCRIPTION
**Pull request by cody-clark**

### CODE REVIEW

**Comment by cody-clark** [comment]
Andrew, I went with the default K8s button styling for consistency, but if you think it needs a stronger call to action, let me know and I'll play with more contrast / hover effects.

**Comment by zparnold** [comment]
screen shot 2018-09-11 at 9 14 29 pm
Just FYI @cody-clark, a hugo shortcode appears to have slipped in to the site

**Comment by cody-clark** [comment]
@zparnold Ah, that was my attempt at documenting the shortcode. I should probably include more context... or is that not even necessary?

**Comment by Phamhien77779** [comment]
EPX8


2018-09-12 12:40 GMT+07:00, Cody Clark <notifications@github.com>:

File: content/en/docs/contribute/style/style-guide.md [hunk 1]
```diff
@@ -263,6 +263,20 @@ The output is:
 **Warning:** Beware.
 {{< /warning >}}
 
+### Katacoda Embedded Live Environment
+
+This button lets users run Minikube in their browser.
```
Comment (chenopis) [inline content/en/docs/contribute/style/style-guide.md]: I would add a little more documentation and description here. Specifically, that this is the Katacoda [Terminal Panel](https://www.katacoda.com/embed/panel) configured to run `minikube start`, and it lets users run Minikube as a panel in their browser so they can follow along while they read the documentation.

Comment (cody-clark) [reply content/en/docs/contribute/style/style-guide.md]: @chenopis done. I've made a proper entry to the style guide.

**Pull request merged**